src/main/java/Application.java

```diff
@@ -97,5 +100,17 @@ public static void main(String[] args) throws AuthenticationException {
         System.out.println(shoppingCartService.getByUser(testShoppingCart));
         shoppingCartService.clear(shoppingCartService.getByUser(testShoppingCart));
         System.out.println(shoppingCartService.getByUser(testShoppingCart));
+
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+        shoppingCartService.registerNewShoppingCart(visitor);
+        shoppingCartService.addSession(movieSession3, visitor);
+        orderService.completeOrder(shoppingCartService.getByUser(visitor).getTickets(), visitor);
+        shoppingCartService.addSession(movieSession, visitor);
+        shoppingCartService.addSession(movieSession4, visitor);
+        orderService.completeOrder(shoppingCartService.getByUser(visitor).getTickets(), visitor);
+        List<Order> ordersHistory = orderService.getOrderHistory(visitor);
+        for (Order singleOrder: ordersHistory) {
+            System.out.println(singleOrder.toString());
```

Comment:
toString() is called automatically here.

Reply:
done